storage/statedb/database.go

```diff
@@ -335,6 +339,17 @@ func NewDatabaseWithCache(diskDB database.DBManager, cacheSize int, daBlockNum u
 	}
 }
 
+func getCacheSize() int {
+	totalPhysicalMem := float32(common.TotalPhysicalMemGB)
+	var cacheSizeGB float32
+	if totalPhysicalMem/4 < 2.5 {
```

Comment:
How about this?
```
	if totalPhysicalMem < 3 {
             // panic
        else if totalPhysicalMem < 16 {
			cacheSizeGB = totalPhysicalMem - 3
	} else {
			cacheSizeGB = totalPhysicalMem  * 12 / 16
	}
```

Reply:
If an panic occures, NewDatabaseWithCache() cannot be accomplished.
Moreover, if totalPhysicalMem is 11~15 it is better to have `totalPhysicalMem * 3 / 4`

I was thinking to change as following:
```
func getTrieNodeCacheSizeMB() int {
	totalPhysicalMem := float64(common.TotalPhysicalMemGB)
	var cacheSizeGB float64
	if totalPhysicalMem/4 < 2.5 { // 2.5GB margin at minimum (0.75GB for NewCache, 1.75GB fore rest)
		cacheSizeGB = math.Max(totalPhysicalMem-2.5, 0)
	} else {
		cacheSizeGB = totalPhysicalMem * 3 / 4
	}
	return int(cacheSizeGB * 1024)
}
```